corehq/apps/commtrack/sms.py

```diff
@@ -124,27 +123,13 @@ def parse(self, text):
             args = args[1:]
 
         action = self.commtrack_settings.action_by_keyword(action_keyword)
-        if action and action.type == 'stock':
```

Comment:
i don't know much about this code but how come we're not checking action.type here anymore?